py-polars/polars/series/series.py

```diff
@@ -4515,6 +4515,11 @@ def apply(
         Series
 
         """
+        # from polars.utils.udfs import warn_on_inefficient_apply
```

Comment:
unintentional?

Reply:
> I think @alexander-beedie meant this as a todo?

Indeed! Probably a slightly bad habit to leave them there commented-out, but the intention is to enable them in the next pass :)